### PR TITLE
README.FreeBSD: update to match the bimonthly FreeBSD status report

### PR DESCRIPTION
**Pull request by wjwithagen**
Signed-off-by: Willem Jan Withagen <wjw@digiware.nl>

### CODE REVIEW

File: README.FreeBSD [hunk 1]
```diff
@@ -1,67 +1,127 @@
...
+	Calling these doest not really work since nostests is not in 
+	/usr/bin, thru /usr/bin/env/nosetests does not work on FreeBSD.
+	test/pybind/test_ceph_argparse.py
+	test/pybind/test_ceph_daemon.py
 
-Things to investigate:
-======================
+Task to do:
+===========
+ -	Build an aumated testplatform that will build ceph/master on
```
Comment (tchaikov) [inline README.FreeBSD]: s/aumated/automated/, s/testplatform/test platform/

File: README.FreeBSD [hunk 2]
```diff
@@ -1,67 +1,127 @@
...
-Things to investigate:
-======================
+Task to do:
+===========
+ -	Build an aumated testplatform that will build ceph/master on 
+	FreeBSD and report the results back to the Ceph developers. This will 
+	increase the maintainability of the FreeBSD side of things. 
+	Developers are signalled that they are using Linux-isms that will not
+	compile/run on FreeBSD Ceph has several projects for this: Jenkins, 
+	teuthology, palpito, ...
```
Comment (tchaikov) [inline README.FreeBSD]: pulpito

File: README.FreeBSD [hunk 3]
```diff
@@ -1,67 +1,127 @@
...
+ -	Compile and test the user space RBD (Rados Block Device).
+
+ -	Investigate and see if an in-kernel RBD device could be developed a la 
+	'ggate'
+
+ -	Investigate the keystore, which could be kernel embedded on Linux an 
+	currently prevents building Cephfs and some other parts.
+
+ -	Scheduler information is not used atm, because the schedulers work 
+	rather different. But a a certain point in time, this would need some
```
Comment (tchaikov) [inline README.FreeBSD]: s/a a/at a/

**Comment by tchaikov** [comment]
just ran it through ispell =)

**Comment by wjwithagen** [comment]
@tchaikov 
'mmmm, that's a tool I haven't used in ages... Guess I'll have to check why my desktop editor is not complaining.
Will Fix.

**Comment by wjwithagen** [comment]
@tchaikov 
Ping...

File: README.FreeBSD [hunk 4]
```diff
@@ -1,67 +1,127 @@
-
 The FreeBSD build will build most of the tools in Ceph.
-Note that the RBD dependant items will not work since FreeBSD does not have RBS
+Note that the RBD dependant items will not work since FreeBSD does not 
+have RBD (yet)
+
+I started looking into Ceph, because the HAST solution with CARP and 
+ggate did not really do what I was looking for. But I'm aiming for 
+running a Ceph storage cluster of storagenodes that are running ZFS.
```
Comment (tchaikov) [inline README.FreeBSD]: storage nodes

File: README.FreeBSD [hunk 5]
```diff
@@ -1,67 +1,127 @@
...
+Endstation would be running bhyve on RBD disk that are stored in Ceph.
```
Comment (tchaikov) [inline README.FreeBSD]: End station

File: README.FreeBSD [hunk 6]
```diff
@@ -1,67 +1,127 @@
...
+ -	All test (but one) run to completion for the current selection of 
+	tools. This is only the case for "My Fork" repository. Some of the 
+	commits need to be pulled into the HEAD
+
+ -	Cmake is now the only way of building Ceph
+ -	Reworked threading/polling code for the simple socket code.
+	Now uses a selfpipe, instead of using an odd shutdown() signaling 
+	Linux feature.
+ -	Modified the EventKqueue code to work around the "feature" that 
+	starting threads descroys the the kqueue handles. Just finshed the
```
Comment (tchaikov) [inline README.FreeBSD]: destroys

File: README.FreeBSD [hunk 7]
```diff
@@ -1,67 +1,127 @@
...
-   thru /usr/bin/env/nosetests does not work on FreeBSD.
-     test/pybind/test_ceph_argparse.py
-     test/pybind/test_ceph_daemon.py
+ -	run-tox-ceph-disk
+	This is the main test that tests the basic Ceph disk install tool.
+	One of the next things to complete porting for.
+ -	ceph-detect-init/run-tox.sh
+	Because the current implementation does not know anything about 
+	FreeBSD rc-init.
+ - 	Tests that make use on nosestests.
```
Comment (tchaikov) [inline README.FreeBSD]: nosetests

**Comment by wjwithagen** [comment]
@tchaikov 
Ping again, reread it and did find a few more.

**Pull request merged**